src/providers/authProvider/permissions.ts

```diff
@@ -26,51 +28,9 @@ const permissions: Record<UserRoles, ResourcePermissions> = {
 
 export default permissions
 
```

Comment:
@TahaKhanAbdalli  - we've removed a large block of code here.

Can you give me a summary of why we've been able to do that?

Reply:
This was being used for getting permissions of the multiple roles assigned to the user. But now as we are using only one role therefore, we don't need to merge permissions.

Reply:
And as the previous code was only sending permissions[role] if only one role was provided therefore, indicating that the removed code was being used for multiple roles 
![image](https://github.com/DeepBlueCLtd/RCO/assets/50602678/a16ed84f-fb9e-4125-b23e-a4f7728725a3)